lib/project_and_version_value.rb

```diff
@@ -83,9 +83,9 @@ def incomplete_project_issues(proj, basis_date)
   # @param [project] proj project object
   # @return [Array] project_id, fixed_version_id
   def project_varsion_id_pair(proj)
-    Issue.cross_project_scope(proj, 'descendants')
-      .where('start_date IS NOT NULL AND due_date IS NOT NULL AND fixed_version_id IS NOT NULL')
-      .joins(:fixed_version).order('effective_date ASC')
+    Issue.cross_project_scope(proj, "descendants")
+      .where("start_date IS NOT NULL AND due_date IS NOT NULL AND fixed_version_id IS NOT NULL")
```

Comment:
Line is too long. [96/80]<br>Place the . on the previous line, together with the method call receiver.